test/automated/chat.test.js

```diff
@@ -2,52 +2,43 @@ const { test } = require('@jest/globals');
 var request = require('supertest');
 request = request('http://127.0.0.1:8080');
 
-const WebSocket = require('ws');
-var ws;
+const registerChat = require('./lib/chat').registerChat;
+const sendChatMessage = require('./lib/chat').sendChatMessage;
 
-const testMessageId = Math.random().toString(36).substring(7);
-const username = 'user' + Math.floor(Math.random() * 100);
+var userDisplayName;
 const message = Math.floor(Math.random() * 100) + ' test 123';
-const messageRaw = message + ' *and some markdown too*';
-const messageMarkdown = '<p>' + message + ' <em>and some markdown too</em></p>'
-const date = new Date().toISOString();
+// const messageRaw = message + ' *and some markdown too*';
+// const messageMarkdown = '<p>' + message + ' <em>and some markdown too</em></p>'
 
 const testMessage = {
-    author: username,
-    body: messageRaw,
-    id: testMessageId,
-    type: 'CHAT',
-    visible: true,
-    timestamp: date,
+  body: message,
+  type: 'CHAT',
 };
 
-test('can send a chat message', (done) => {
-    ws = new WebSocket('ws://127.0.0.1:8080/entry', {
-    origin: 'http://localhost',
-});
 
-    function onOpen() {
-        ws.send(JSON.stringify(testMessage), function() {
-            ws.close();
-            done();
-        });
-    }
+test('can send a chat message', async (done) => {
```

Comment:
As far as I've seen tests have always run in order, so I've never seen any issues from it.  But I'm no expert in Jest.  Just for clarification I checked with @gingervitis and she believes they always run in order.  This could be updated in the future if we want to spend some time grouping them together.  But all the tests currently work this way, and they're not broken now and are performing as expected.